tox.ini

```diff
@@ -67,33 +60,28 @@ commands =
     docker-compose rm --force
 
 [testenv:paasta_itests_inside_container]
-basepython = python2.7
 envdir=/tmp/
 setenv =
     DOCKER_COMPOSE_PROJECT_NAME = paastatools_inside_container
 changedir=paasta_itests/
-deps =
-    {[testenv]deps}
-    behave==1.2.4
-    behave-pytest==0.1.1
-    # This .whl is created in yelp_package/dockerfiles/trusty/Dockerfile
-    /root/mesos.executor-1.0.1-py27-none-any.whl
-    /root/mesos.scheduler-1.0.1-py27-none-any.whl
-    /root/mesos.native-1.0.1-py27-none-any.whl
 commands =
+    {toxinidir}/bin/venv-update venv= {envdir} install= -r {toxinidir}/requirements-dev.txt -e {toxinidir} \
```

Comment:
I've realized this is a problem when running `make_trusty` and `make_xenial` so I'll have to figure out why this is the case (or hardcode it everywhere, which I don't really want to do).